com.unity.robotics.urdf-importer/Runtime/MeshProcessing/StlWriter.cs

```diff
@@ -0,0 +1,190 @@
+/*
+© Siemens AG, 2018
+Author: Suzannah Smith (suzannah.smith@siemens.com)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+<http://www.apache.org/licenses/LICENSE-2.0>.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace RosSharp.Urdf
+{
+    public class StlWriter
```

Comment:
nit: I feel this script is more useful than just for URDF-importer. It is kind of related to Devin's question that whether we want to have a core package that the URDF-Importer can also depend on? It is not a question for this PR, but just a marker for the thought.

Reply:
Agree, we can move it once the need arises and we have a core.